docs/vereisten/traceerbare-besluitvorming.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Traceerbare besluitvorming
+toelichting: Als een besluit of beslissing tot stand is gekomen door of met behulp van een algoritme of AI-systeem, dan moet traceerbaar zijn hoe het algoritme of het AI-systeem tot de output is gekomen. 
+status_vereiste:
+- geldend
+levenscyclus:
+- ontwerp
+- ontwikkelen
+- verificatie-en-validatie
+- monitoring-en-beheer
+- implementatie
+bouwblok:
+- Transparantie
+  
+---
+
+<!-- tags -->
+## Vereiste
+
+De werking en output van een algoritme of AI-systeem moet traceerbaar zijn als deze bijdraagt aan de totstandkoming van besluiten en beslissingen. 
+
+## Toelichting
+
+Als een besluit of beslissing tot stand komt met behulp van een algoritme of AI-systeem, dan is het van belang dat de output traceerbaar is. Dit is van belang zodat de output bij het nemen van een besluit of beslissing of achteraf kan worden gecontroleerd. 
```

Comment:
Kunnen we nog uitleggen wat 'traceerbaar' betekent? Moeten de uitkomsten reproduceerbaar zijn? Moet je tussenresultaten loggen?

Reply:
Goede vragen, daar heb ik niet meteen een antwoord op. Op de NORA online wordt een definitie gegeven voor traceerbaarheid: https://www.noraonline.nl/wiki/Traceerbaarheid. Ik zie deze wel helemaal opgaan onder transparantie. We kunnen het ook (in de volgende release) daar een duidelijkere plek geven. We hebben nu een aantal vereisten die over transparantie gaan (obv AI-verordening, AVG etc.), maar dit is in zijn algemeenheid ook van belang.